docs/getting-started/install_capi_operator.md

```diff
@@ -4,4 +4,55 @@ sidebar_position: 3
 
 # Install CAPI Operator
 
-This section needs to be filled out.
\ No newline at end of file
+## Installing CAPI and providers
+
+CAPI and desired CAPI providers could be installed using helm based installation for [`Cluster API Operator`](https://github.com/kubernetes-sigs/cluster-api-operator).
+
+### Cluster API operator path
+
+For example, to install latest version of CAPI + docker provider, first, the helm repo should be added with:
+```bash
+helm repo add capi-operator https://kubernetes-sigs.github.io/cluster-api-operator
+helm repo update
+```
+and installed together with [cert-manager](https://github.com/cert-manager/cert-manager) using:
+```bash
+helm install capi-operator capi-operator/cluster-api-operator
```

Comment:
Could we include the **set** to make sure v1.4.4 of the core provider is installed.

Reply:
Depending on that https://github.com/rancher-sandbox/rancher-turtles/pull/109, some values might change and there is no need to install CAPI Operator manually, it's included in extension chart

Reply:
I’ve updated everything accordingly as https://github.com/rancher-sandbox/rancher-turtles/pull/109 was merged.